odbc/src/api/data.rs

```diff
@@ -62,6 +62,18 @@ fn i64_to_bit(i: i64) -> Result<(u8, Option<ODBCError>)> {
     }
 }
 
+fn is_string_fraction_precision_micro(s: &str) -> bool {
```

Comment:
nit: `is_x` is generally used in a chainable operation (much like other `verb_type` methods. I think it may be clearer when reading the code to start with the type, then what you're checking, i.e. `string_contains_fractional_precision_micros` 

```suggestion
fn string_contains_fractional_precision_micros(s: &str) -> bool {
```

Reply:
Makes a lot of sense, and is definitely a function of me going Python -> Rust. Thanks for pointing this out!